tox.ini

```diff
@@ -84,9 +84,7 @@ commands =
 [testenv:regen]
 changedir = doc/en
 basepython = python3
-passenv = SETUPTOOLS_SCM_PRETEND_VERSION
-# TODO: When setuptools-scm 5.0.0 is released, use SETUPTOOLS_SCM_PRETEND_VERSION_FOR_PYTEST
-# and remove the next line.
+passenv = SETUPTOOLS_SCM_PRETEND_VERSION_FOR_PYTEST
 install_command=python -m pip --use-deprecated=legacy-resolver install {opts} {packages}
```

Comment:
The fix should enable the removal of the `--use-deprecated=legacy-resolver` here